### ISSUE TITLE
Can't restore on Mac from a backup created on WIndows in 2.0.0.80

### ISSUE THREAD

**Issue by davidwbryant** [duplicati/duplicati#1302]
I made some backups on Windows 7 and wanted to test them on a different computer. The one available happened to be a Mac running 10.10.2. 

Actually the version on the Mac is later than 2.0.0.80 - it came from latest.zip on 16.2.2015.

Using Restore from Target from the web interface, I can connect to OneDrive successfully, the database builds, and a Restore Files & folders box appears. This shows the expected backup date, but not the folders that the backup contains. Instead there is a single folder icon with no name, and a check box beside it.

If I check the box and click next, a pop-up appears to say "You must select at least one path to restore".

The same backup restores successfully on a Windows 7 system (not the same one it was created on).

This looks similar to issue #1194, but I think not the same as I can restore my backup on a Windows 7 system.

I'm not absolutely sure whether creating a backup on Windows and restoring on Mac is a supported use case, but it was possible with 1.3.4
##

**Comment by ingul-buz** [comment]
I'm wondering if this issue has any relation to the issue I opened a few weeks ago (#1264). Maybe not, as I can always see the new/updated files that were backed up during each individual backup, but *only* those and never all of the files at once.

There are some similarities though in that this happens only when doing a clean restore to another machine (i.e. the local database has to be reconstructed). In my case both of them are Windows 7 machines. This started to happen at some point after 2.0.0.73 and is still present in 2.0.0.82.

**Comment by davidwbryant** [comment]
Following up from the comment by ingul-buz, I installed 2.0.0.73 on the Mac, but that behaves in the same way as the later versions, so probably these aren't the same issue.

I also tried downloading the files to the Mac and then restoring from the local drive but that didn't make any difference.

I made a small test backup to OneDrive from the Mac and was able to restore it ok. I also deleted the backup job on the Mac and was still able to restore from the files on OneDrive.

It looks as though this is an issue with reconstructing the local database on Mac from a remote database that was created on a Windows system
